### PR TITLE
Add acm cert

### PR DESCRIPTION
**Pull request by geekbass**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -253,5 +253,8 @@ module "dcos-lb" {
   name_prefix                        = "${var.name_prefix}"
   disable_masters                    = "${var.lb_disable_masters}"
   disable_public_agents              = "${var.lb_disable_public_agents}"
+  masters_acm_cert_arn               = "${var.masters_acm_cert_arn}"
+  masters_internal_acm_cert_arn      = "${var.masters_internal_acm_cert_arn}"
+  public_agents_acm_cert_arn         = "${var.public_agents_acm_cert_arn}"
```
Comment (bernadinm) [inline main.tf:258]: Cool I see this is also added here as well https://github.com/dcos-terraform/terraform-aws-lb-dcos/blob/release/v0.2/variables.tf#L56-L69

Comment (bernadinm) [reply main.tf:258]: And this version is tagged with `0.2.0` and is available for use. Just wanted to check. LGTM

**Review by bernadinm — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -41,102 +41,100 @@ Known Issues
...
+| public\_agents.private\_ips | Public Agent instances private IPs |
+| public\_agents.public\_ips | Public Agent public IPs |
+| security\_groups.admin | This is the id of the admin security_group that the cluster is in |
+| security\_groups.internal | This is the id of the internal security_group that the cluster is in |
+| vpc.cidr\_block | This is the cidr_block of the VPC the cluster is in |
 | vpc.id | This is the id of the VPC the cluster is in |
-| vpc.main_route_table_id | This is the id of the VPC's main routing table the cluster is in |
-| vpc.subnet_ids | This is the list of subnet_ids the cluster is in |
+| vpc.main\_route\_table\_id | This is the id of the VPC's main routing table the cluster is in |
+| vpc.subnet\_ids | This is the list of subnet_ids the cluster is in |
```
Comment (bernadinm) [inline README.md]: It looks like this recent commit is attempting to escape underscores

**Review by bernadinm — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (geekbass) [reply README.md]: We have moved to latest version of terraform-docs for this release per @fatz

Comment (fatz) [reply README.md]: @bernadinm y which makes sense as per markdown you change _textstyle_ with \_

Comment (sebbrandt87) [reply README.md]: `terraform-docs` in this version is also already like that integrated into the `terraform-jenkins` container image ;)
So the CI will use the same for testing and later creation of `README.md` ;)

**Pull request merged**